build.xml

```diff
@@ -204,37 +208,42 @@
     </or></not></condition></fail>
 
     <resources id="_jvm11_arg_items">
+        <!-- BELOW IS A HACK TO RUN TESTS with newer JDKs. We need to decide whether we add CASSANDRA-18179 and get
+        rid of CASSANDRA_USE_JDK11 and use-jdk11 before 5.0-->
```

Comment:
That is a separate topic in my opinion. Maybe we open a separate issue if we want to do that?
Also, we should consider it will diverge from the Apache 4.0 and there will be conflicts on rebase.

Reply:
We will rather not be rebasing, please remove `CASSANDRA_USE_JDK11` and `use.jdk11` - they are useless

Reply:
also don't bother about rebasing - Cassandra 4.0 does not seem to have much more changes in `build.xml` beyond our current base (4.0.11 IIRC)

Reply:
btw. could you simply create another section with `_jvm22_arg_items`?

Reply:
Ok, I will wrap up a patch that can be committed today, this was hack for experimenting/testing